src/com/projectkorra/projectkorra/attribute/AttributeModifier.java

```diff
@@ -9,8 +9,10 @@
 
 public class AttributeModifier {
 	
```

Comment:
Again, if something is unused we want to at least see that and have it pointed out to us.